cypress/e2e/app_test/stepper.cy.js

```diff
@@ -0,0 +1,23 @@
+import { HEADERS, URL  } from "../../support/constants";
+import { LOCATORS} from "../../support/locators";
+
+describe ('Stepper page', () => {
+
+    it ('First test',()=> {
+        cy.visit(URL.Stepper);
+        cy.get(LOCATORS.Stepper.StepContent)
+            .should('contain.text', HEADERS.StepContent1);
+        cy.get(LOCATORS.Stepper.btnNextTopBlock).click();
+        cy.get(LOCATORS.Stepper.StepContent)
+        .should('contain.text', HEADERS.StepContent2);
+            cy.get(LOCATORS.Stepper.btnNextTopBlock).click();
+        cy.get(LOCATORS.Stepper.StepContent)
+            .should('contain.text', HEADERS.StepContent3);
+        cy.get(LOCATORS.Stepper.btnNextTopBlock).click();
+        cy.get(LOCATORS.Stepper.StepContent)
+            .should('contain.text', HEADERS.StepContent4);
```

Comment:
Small improvement to your code, to not use DRY
```suggestion
        for (const key in HEADERS) {
            if (HEADERS.hasOwnProperty(key)&& key !=='HorizontalForm' && key !=='EnterYourName') {
              cy.get(LOCATORS.Stepper.StepContent).should('contain.text',  HEADERS[key]);
           key !== 'StepContent4'?cy.get(LOCATORS.Stepper.btnNextTopBlock).click():"Finish"
            }
          }
```